### PR TITLE
Add renovate.json5

### PR DESCRIPTION
**Pull request by j-low**
# Renovate // Flesh out `renovate.json5` with broader configuration.

Add additional configuration to `renovate.json5` mirroring much of what has been implemented in the managed service repo.

#### Checklist
- [ ] I have written or updated test for the changes I made
- [ ] I have updated the README of the package I'm working in to reflect my changes
- [ ] I have added or updated Storybook if appropriate for my changes

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      },
+    ],
+  },
+
+  // Do we need to add this here? Unsure if Lerna package updates will occur
+  // properly without it.
+  // Relevant documentation:
+  // - https://docs.renovatebot.com/presets-monorepo/
+  // - https://docs.renovatebot.com/configuration-options/#matchsourceurlprefixes
+  matchSourceUrlPrefixes: ["https://github.com/lerna/lerna"],
```
Comment (j-low) [inline renovate.json5:60]: @nathanstilwell we were a bit unsure whether this piece of config was necessary.  Our thought was that perhaps the Lerna repo's subpackages may not be properly detected without it (and we get a broken Lerna dependency in the UI repo) however that may be based on a misreading of the docs.  Given your deeper knowledge of monorepos, we'd benefit from your insight.

Comment (laurenbarker) [reply renovate.json5:60]: It looks like `matchSourceUrlPrefixes` groups updates, but it's unclear how a lerna package group would interact with the defined groups above. Easiest is to merge it and see what Renovate does.

**Review by laurenbarker — APPROVED** [review]
LGTM! It might take merging it and seeing the output to get it working in an ideal way. No risk and merging and iterating. Thanks for setting this up!

File: renovate.json5 (hunk 1 above)
Comment (nathanstilwell) [reply renovate.json5:60]: I agree with Lauren that we should probably merge and see what happens.

**Review by nathanstilwell — APPROVED** [review]
LGTM, let's give it a shot.

**Comment by j-low** [comment]
Awesome, TFTR @laurenbarker & @nathanstilwell.  Happy to merge and see what happens 😂

**Pull request merged**